fordel-domene/src/main/java/no/nav/foreldrepenger/mottak/task/TilJournalføringTask.java

```diff
@@ -16,18 +16,19 @@
 import no.nav.foreldrepenger.mottak.person.PersonInformasjon;
 import no.nav.vedtak.exception.TekniskException;
 import no.nav.vedtak.felles.prosesstask.api.ProsessTask;
-import no.nav.vedtak.felles.prosesstask.api.ProsessTaskRepository;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskTjeneste;
+import no.nav.vedtak.felles.prosesstask.api.TaskType;
 
 /**
  * <p>
  * ProssessTask som utleder journalføringsbehov og forsøker rette opp disse.
  * </p>
  */
 @ApplicationScoped
-@ProsessTask(TilJournalføringTask.TASKNAME)
+@ProsessTask("fordeling.tilJournalforing")
```

Comment:
Er det ikke bedre slik det var før? At man har taskname i static variabel, så slipper man "TaskType.forProsessTaskHandler(TilJournalføringTask.class).value()"?

Reply:
Dette er delvis for å kompensere for bortfall av task-type-tabellen.
Pluss at jeg ville bli kvitt public-bruken i public static String TASK_NAME 

Dersom man bruker tasknavn ofte i exceptions kan man ha en "static String TASK_NAME" som brukes i annoteringen

Reply:
Master for tasknavn skal være det som er oppgitt i annoteringen

Reply:
Men har oppdatert. Det er litt scoping-issues